hackclub.com.yaml

```diff
@@ -282,6 +282,10 @@ ciudalcampo:
 - ttl: 1
   type: CNAME
   value: code-sekcc.github.io.
+coins:
+- ttl: 1
+  type: A
+  value: 173.208.140.124
```

Comment:
It says " Nothing here yet :/ ". Is this intentional for now?

Reply:
@lukec11

Reply:
Yep, once this is merged nginx will auto route this to the correct spot @hkatzdev .